content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Tips for Submitting a Video Abstract to Posit Conf 2025"
```

Comment:
```suggestion
title: "Tips for Submitting a Video Abstract to Posit Conf"
```

should this be dated 2025 or more general?

Reply:
Let's do general actually so we can recycle